src/librustc_lexer/src/lib.rs

```diff
@@ -1,3 +1,16 @@
+//! Basic Rust lexer.
```

Comment:
```suggestion
//! Low-level Rust lexer.
```
Perhaps this is a better word for what you were trying to say.